src/components/ApplicationTile/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { useTranslation } from 'react-i18next';
+import classNames from 'classnames';
+import Text from '../Text';
+import styles from './styles.css';
+import getFormattedTimeDifference from '~/utils/getFormattedTimeDifference';
+
+interface Props {
```

Comment:
please change name to ApplicationTileProps

Reply:
I'm not sure it is necessary since we have only one component in this file.
I would say it is easier to read the code if every file contains the same interface named `Props`

Reply:
Ok, you're right